com/Tharani_Dharan/Fibonacci.java

```diff
@@ -0,0 +1,22 @@
+package com.Tharani_Dharan;
+
+import java.util.Scanner;
+
+public class Fibonacci {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.print("Enter a number : ");
+        int num = sc.nextInt();
+        int a = 0;
```

Comment:
nitpick: Can we name the variables in such a way that it would make sense? Please rename it to something like: 
`previous`, `next`, etc